### DISCUSSION TITLE
HOMEBRIDGE TUYA WEB - IRRIGATION VALVES DEVICES

### DISCUSSION THREAD

**Discussion by jbarea54** [milo526/homebridge-tuya-web#422] [Q&A]
hello to everyone.

It's my first time asking and, also, it's my first time working with the plugin Tuya Web.

I have installed the plugin and is working perfectly with all the devices except with the irrigation valves.

I'm reading several information about this, but is hard for me understand how I can proceed.

Can somebody help me?

Thanks in advance.

**Comment by milo526** [comment]
Irrigation devices are not, cannot, and therefor will not, be implemented by this plugin.
This is a limitation imposed by Tuya which I cannot bypass sadly, please take a look at Tuna's official homebridge plugin, with that it should work.

**Reply by jbarea54** [reply]
Thank you so much milo526. 
Wich one? Tuna's? I don't find this plugin. Sorry if I'm a redneck but I'm starting using homebridge...
Thanks in advance.

**Reply by milo526** [reply]
https://github.com/tuya/tuya-homebridge
